modules/javafx.graphics/src/test/java/test/javafx/scene/NodeTest.java

```diff
@@ -2023,4 +2021,36 @@ public interface StubRectAccessor {
         }
 
     }
+
```

Comment:
Since you changed the copyright year on the other files, you might want to do this one too. There is a script that will do it anyway, so it doesn't matter.

Reply:
Done.